internal/dnsanalyzer/dnsanalyzer.go

```diff
@@ -0,0 +1,108 @@
+package dnsanalyzer
+
+import (
+	"net"
+
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
+)
+
+type rawLookup struct {
+	query *layers.DNS
+	reply *layers.DNS
+}
+
+type DNSAnalyzer struct {
+	lookups     map[uint16]*rawLookup
```

Comment:
Do we actually need to save these?

Reply:
I was going to use them in the future, but its better to keep unused code out.